### PR TITLE
Add STM32C0 support

### PR DESCRIPTION
**Pull request by xanthio**
This Add support for STM32C0 range MCU

### CODE REVIEW

File: cmake/stm32/c0.cmake [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "C011.."
+    "C031.."
+)
+set(STM32_C0_RAM_SIZES 
+     6K
+    12K
+)
+set(STM32_C0_CCRAM_SIZES 
+     0K  0K
+     0K  0K
```
Comment (Hish15) [inline cmake/stm32/c0.cmake:15]:
```suggestion
     0K 
     0K 
```

**Review by Hish15 — CHANGES_REQUESTED** [review]

File: cmake/stm32/c0.cmake [hunk 2]
```diff
@@ -0,0 +1,25 @@
+set(STM32_C0_TYPES 
+    C011xx
+    C031xx
+)
+set(STM32_C0_TYPE_MATCH 
+    "C011.."
+    "C031.."
```
[the remark this answers is not in the document]
Comment (xanthio) [reply cmake/stm32/c0.cmake]: resolved by commit [1e6bcd9](https://github.com/ObKo/stm32-cmake/pull/331/commits/1e6bcd93318a8283243bab4d5eabad0a8b9d396b)

File: cmake/stm32/c0.cmake (hunk 1 above)
Comment (xanthio) [reply cmake/stm32/c0.cmake]: Resolved by commit [58dfa5b](https://github.com/ObKo/stm32-cmake/pull/331/commits/58dfa5b832d051841fb34cc3bf7cf8720ddbc471)

**Review by Hish15 — APPROVED** [review]
Good quality on this PR!

**Comment by Hish15** [comment]
@atsju, I let you review and merge (in case I missed something).

**Review by atsju — APPROVED** [review]
looks excellent. Thank you very much

**Comment by Hish15** [comment]
@xanthio and @atsju, we missed something :)
@xanthio, can you please open a new PR to notify this addition into the Readme.md file please ? 
At least 
https://github.com/ObKo/stm32-cmake/blob/b34988ee4a32002a9f6160acb118ef93abe9996a/README.md?plain=1#L6
and 
https://github.com/ObKo/stm32-cmake/blob/b34988ee4a32002a9f6160acb118ef93abe9996a/README.md?plain=1#L59

**Pull request merged**